Sources/config.hpp

```diff
@@ -0,0 +1,14 @@
+#ifndef NETKET_CONFIG_HPP
+#define NETKET_CONFIG_HPP
+
+#if defined(WIN32) || defined(_WIN32)
+#define NETKET_EXPORT __declspec(dllexport)
+#define NETKET_NOINLINE __declspec(noinline)
+#define NETKET_FORCEINLINE __forceinline inline
+#else
```

Comment:
Where are these used?

Reply:
Not anymore :) Thanks for pointing it out!